ow_regolith/include/DigStateMachine.h

```diff
@@ -0,0 +1,176 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Defines a finite state machine and associated states that estimate the phase
+// of a digging operation performed with the lander scoop.
```

Comment:
Minor: A class diagram (e.g. UML) would be helpful in documenting this class.

Reply:
I don't know UML, but I wouldn't mind an opportunity to learn it. Where would this diagram be kept? Confluence?

Reply:
Since the code is public, its documentation may as well be too.  I'd suggest an Appendix in the user guide (wiki).  I've put a UML of autonomy in that chapter.